WrightTools/artists/_interact.py

```diff
@@ -201,7 +203,8 @@ def interact2D(data, xaxis=0, yaxis=1, channel=0, local=False, verbose=True):
     current_state.zi = zi
     clim = get_clim(channel, current_state)
     xi, yi, zi = pcolor_helper(xaxis.points[:, None], yaxis.points[None, :], zi)
-    obj2D = ax0.pcolormesh(xi, yi, zi, cmap=cmap, vmin=clim[0], vmax=clim[1])
+    plt.sca(ax0)
+    obj2D = plt.pcolormesh(xi, yi, zi, cmap=cmap, vmin=clim[0], vmax=clim[1])
```

Comment:
Why is this necessary? ax.pcolormesh should be able to handle bare arrays as well as data objects.
If it does not, it should be fixed to do so as that is intended behavior

Reply:
@p770193 (bumping because this got collapsed in `black` commit)

Reply:
I also hesitate to have unexpected side effects within this function (the axes pointed to by plt could be changed for a user)